aospy/calc.py

```diff
@@ -254,7 +253,7 @@ def _get_input_data_paths_gfdl_dir_struct(self, name, data_in_direc,
                                                   self.intvl_out,
                                                   self.data_in_start_date[n].year,
                                                   self.data_in_dur[n]))
-                 for year in range(start_year, end_year + 1)]
```

Comment:
Sorry about this (I didn't mean to include this in the PR) -- this is leftover from how I defined the ```default_date_range``` in my personal Run objects; I would put the end date as January 1st the following year (e.g. if I wanted years [2000, 2004] inclusive I would put my date range as ("2000-01-01", "2005-01-01")).  

As you point out though, if one were to specify a non-yearly interval (like "2000-01-01" to "2004-02-01"), then this logic would fail to retrieve the necessary data for January 2004 (not to mention it isn't consistent with the inclusive-on-both-ends logic in ```_get_time()``` within timedate.py).  For this reason I will stop being stubborn and fix my Run objects to be consistent with the current aospy conventions.